ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_StateSpecs.rc

```diff
@@ -9,8 +9,16 @@ category: IMPORT
 #----------------------------------------------------------------------------------------
  DU_SRC     | 1      | xy   | N   |      | erod - dust emissions
  FRLAKE     | 1      | xy   | N   |      | fraction_of_lake
+ FRSNOW     | 1      | xy   | N   |      | surface_snow_area_fraction
```

Comment:
Doesn't this impose a requirement on users of the default DU scheme to provide these variables?    I think an additional parameter is  needed in the table to make these conditional.

Reply:
Conditional import variables have been implemented with commit 261a4ac7a4d08fd0cee7050ed3c1b2f9817bd235